time_stamper_ros/include/Node.h

```diff
@@ -1,13 +1,17 @@
 #pragma once
 #include <iostream>
+#include <string>
+
 #include <ros/ros.h>
+#include <dynamic_reconfigure/server.h>
 
 #include "sysfs/SysfsGpio.h"
 #include "sysfs/SysfsPwm.h"
+#include "time_stamper_ros/LedConfig.h"
 
 enum LedMode {
-  FPS,
-  EXPOSURE
+  FPS = 0,
+  EXPOSURE = 1
 };
```

Comment:
I'd prefer class enums, see https://www.geeksforgeeks.org/enum-classes-in-c-and-their-advantage-over-enum-datatype/